docs/pages/eas-workflows/jobs.mdx

```diff
@@ -3,105 +3,137 @@ title: Jobs
 description: Learn which pre-packaged and custom jobs you can run with EAS Workflows.
 ---
 
-EAS Workflows sequence jobs together to automate your development and release processes. You can run two types of jobs: pre-packaged jobs and custom jobs.
+EAS Workflows sequence jobs together to automate your development and release processes. You can run two types of jobs: pre-packaged jobs and custom jobs. Pre-packaged jobs use a `type` property to specify the job type. Custom jobs run custom code that you define in the workflow configuration.
```

Comment:
This will become slightly less accurate when we introduce "Custom Builds within a Workflow", in which case a `type: build` pre-packaged job suddenly runs custom code, but I think it's ok. We're going for the general understanding. 👍